libraries/app/database_api.cpp

```diff
@@ -922,124 +925,124 @@ std::map<std::string, full_account> database_api_impl::get_full_accounts( const
          acnt.cashback_balance = account->cashback_balance(_db);
       }
 
-      uint64_t api_limit_get_full_accounts = _app_options->api_limit_get_full_accounts;
+      uint64_t api_limit_get_full_accounts_lists = _app_options->api_limit_get_full_accounts_lists;
 
       // Add the account's proposals
       auto  required_approvals_itr = proposals_by_account._account_to_proposals.find( account->id );
       if( required_approvals_itr != proposals_by_account._account_to_proposals.end() )
       {
-         acnt.proposals.reserve( std::min(required_approvals_itr->second.size(), api_limit_get_full_accounts) );
+         acnt.proposals.reserve( std::min(required_approvals_itr->second.size(), api_limit_get_full_accounts_lists) );
+         if(required_approvals_itr->second.size() > api_limit_get_full_accounts_lists)
+            acnt.more_data_available.proposals = true;
          for( auto proposal_id : required_approvals_itr->second )
          {
             acnt.proposals.push_back( proposal_id(_db) );
-            if(acnt.proposals.size() >= api_limit_get_full_accounts) {
-               acnt.more_data_available.proposals = true;
+            if(acnt.proposals.size() >= api_limit_get_full_accounts_lists)
                break;
-            }
          }
       }
 
       // Add the account's balances
       const auto& balances = _db.get_index_type< primary_index< account_balance_index > >().get_secondary_index< balances_by_account_index >().get_account_balances( account->id );
```

Comment:
While you're on it, can you please wrap this long line? Thanks.

Reply:
done at https://github.com/bitshares/bitshares-core/pull/1749/commits/65d518021bd98250dcb928e29bae3b412072ed57